frontend/src/views/ProfileView.vue

```diff
@@ -1,10 +1,79 @@
 <template>
 	<div class="page_box_wrapper">
 		<div class="page_box">
-			<h1>This is a profile page</h1>
+			<VerticalAvatarAndUserName
+				profile_picture="/test-profile.png"
+				:profile_name="this.user.name"
+			/>
+			<OverviewWithMidline :data-array="dataArray" />
 		</div>
 	</div>
 </template>
 
+<style scoped></style>
+
 <script lang="ts">
-</script>
\ No newline at end of file
+import { defineComponent } from 'vue';
+import VerticalAvatarAndUserName from '@/components/user-info/VerticalAvatarAndUserName.vue';
+import OverviewWithMidline from '@/components/overviews/OverviewWithMidline.vue';
+import { ref, onMounted, type Ref, defineProps } from 'vue';
+import type { OverviewArray } from '@/types/OverviewArray';
+import { getRequest } from '@/utils/apiRequests';
+import type { User } from '../types/User';
+
+let dataArray: Ref<OverviewArray[]> = ref([]);
+let user: User = {} as User;
+
+export default defineComponent({
+	name: 'ProfileView',
+	components: {
+		OverviewWithMidline,
+		VerticalAvatarAndUserName,
+	},
+	props: {
+		profile_id: String,
+	},
+	data() {
+		return {
+			user: {} as User,
+		};
+	},
+	async created() {
+		this.user = (await getRequest(`users/${this.profile_id}`)).data;
+		// await useUserStore().refreshAllUsers();
+		// this.user = useUserStore().allUsers.filter((user: User) => (Number(user.id) === Number(this.profile_id)))[0];
+		if (!this.profile_id) {
+			dataArray.value = [
```

Comment:
if you change the dataArray to be intanciated in the `data()` then dataArray.value becomes this.dataArray.